distribution/docs/src/main/resources/content/_managing/attributes-added-on-ingest.adoc

```diff
@@ -0,0 +1,48 @@
+:title: Attributes Added on Ingest
+:type: dataManagement
+:status: published
+:parent: Automatically Added Metacard Attributes
+:summary: How attributes are automatically added to metacards during ingest.
+:order: 0
+
+== {title}
+
+During ingest, a metacard is first created and populated by parsing the ingested resource with an <<_attributes_added_by_input_transformers,input transformer>>.
+Then <<_attributes_added_by_attribute_injection,attributes are injected>> into the metacard, followed by <<_attributes_added_by_attribute_overrides,attribute overrides>>.
+Finally the metacard is passed through a series of <<_attributes_added_by_pre_authorization_plugins,pre-authorization plugins>> and <<_attributes_added_by_pre_ingest_plugins,pre-ingest plugins>>.
+
+.Ingest Attribute Flow
+[ditaa,ingest-attribute-flow]
+....
++----------------------------------------------+
+| /----------------\  |                    cCCC|
+| |     Ingest cDEF|  |                        |
+| \----------------/  |                        |
+|                     v                        |
+| /------------------------------------------\ |
+| |c369        Input Transformers            | |
+| \------------------------------------------/ |
+|                     |                        |
+|                     v                        |
+| /------------------------------------------\ |
+| |c369        Attribute Injection           | |
+| \------------------------------------------/ |
+|                     |                        |
+|                     v                        |
+| /------------------------------------------\ |
+| |c369        Attribute Overrides           | |
+| \------------------------------------------/ |
+|                     |                        |
+|                     v                        |
+| /------------------------------------------\ |
+| |c369     Pre Authorization Plugins        | |
+| \------------------------------------------/ |
+|                     |                        |
+|                     v                        |
+| /------------------------------------------\ |
+| |c369        Pre Ingest Plugins            | |
+| \------------------------------------------/ |
+|                     |                        |
+|                     v                        |
++----------------------------------------------+
```

Comment:
✏️ This flow chart does not seem to be very useful. A flow chart should show a branching based on certain inputs. This just shows a list of the operations on ingest, which I think is well covered by the preceding paragraph. You _could_ include the different ingest methods at the very top and show the steps used to determine the right InputTransformer (and the differences between ingest methods). Other than that, I'm not actually seeing a lot of value in having the flow chart.

Reply:
Now that I look at it, I agree with you. The flowchart just repeats what is said in the paragraph above it.

My personal opinion is that it should be removed. I am open to replacing it with another chart, I'm just not sure what that chart would cover.

@brjeter good idea on the different ingest methods as inputs, but the methods are just too similar. The only real difference is how they decide their input transformers, but even that process is super similar between methods.

Reply:
How about turning the paragraph into a list of steps? That would provide the same clear visual ordering as the flowchart without the redundancy

Reply:
Not a bad idea. Then the above section could do a **very** high level overview, and the new list of steps would provide the flow of ingest and links for more information.